tests/conftest.py

```diff
@@ -935,9 +935,8 @@ def _load_project(
         loader.PluginModuleFinder.reset()
         project = Project(self.project_dir, autostd=autostd, main_file=main_file, venv_path=self.env)
         Project.set(project)
-        project.use_virtual_env()
         self._patch_process_env()
```

Comment:
is this still required now that we have the project taking the right venv_path parameter?